### ISSUE TITLE
Initial Commands?

### ISSUE THREAD

**Issue by fusinstruments** [AdamCarrera/UltraHydro#12]
Should we be putting these commands into a CMD prompt, or Power shell prompt?

**Comment by fusinstruments** [comment]
All the initial Anaconda installation packages worked with the exception of Pyside 2. While using the CMD prompt and attempting to install it we observed a consistent failure. The CMD prompt returned, "Failed with initial frozen solve, retrying with flexible solve."

**Comment by adam-smashr** [comment]
I think I was able to recreate the issue on my end. On the image you sent me, the powershell prompt says that you are in the base environment. `(base) PS C:\Users\rajiv>`

If I attempt to install pyside2 into the base environment, I get a similar error, the prompt retries with flexible solve and then gets stuck. 

My fix is to create a different environment to hold all of our packages `conda create --name some_name python=3.8`.  The base environment of conda has a lot more packages than a fresh environment. I've attached an image below of trying to install it on the base environment and a fresh environment.

![image](https://user-images.githubusercontent.com/6711239/113516420-d9885180-953f-11eb-805f-cc099de45065.png)
